### PR TITLE
Remove anchor tags from DM sections

### PR DESCRIPTION
**Pull request by rssag**
introduced some labels for static template IDs, removed some anchors that were never referenced or identical to the Hugo auto-generated IDs.

There's two more anchors in cloud remote access which could be removed, but they are referenced in the Edge section.
Since it's unclear what happens to Edge currently, I did not touch them.

### CODE REVIEW

File: content/cloud-remote-access/using-cloud-remote-access-bundle/endpoints.md [hunk 1]
```diff
@@ -16,8 +16,7 @@ The "endpoint" is the IP address and port of the VNC, SSH or Telnet server runni
 To be able to configure an endpoint, you need ADMIN permission for "Remote access" and "Device control". To read data, a READ permission is sufficient. For more information on permissions, refer to [Managing permissions](/standard-tenant/managing-permissions/).
 {{< /c8y-admon-info >}}
 
-<a name="adding-remote-access-endpoints-via-vnc"></a>
-#### To add a remote access endpoint via VNC
+#### To add a remote access endpoint via VNC {#adding-remote-access-endpoints-via-vnc}
```
Comment (BeateRixen) [inline content/cloud-remote-access/using-cloud-remote-access-bundle/endpoints.md:19]: Wouldn´t it make sense to change this ID to the actual heading content? Also below.

File: content/device-management-application/monitoring-and-controlling-devices-bundle/connection-monitoring.md [hunk 2]
```diff
@@ -49,14 +49,14 @@ It is always green if the device is connected, even without data.
 Connection monitoring is not real time. This means that the displayed connection status will not change immediately after switching off a device. Depending on the used protocol for push connection monitoring this can take a couple of minutes.
 {{< /c8y-admon-info >}}
 
-<a name="maintenance-mode"></a> **Maintenance mode**
+**Maintenance mode**
```
Comment (BeateRixen) [inline content/device-management-application/monitoring-and-controlling-devices-bundle/connection-monitoring.md:52]: As this is not a heading, is this anchor not referred to?

**Review by BeateRixen — APPROVED** [review]

File: content/cloud-remote-access/using-cloud-remote-access-bundle/endpoints.md (hunk 1 above)
Comment (rssag) [reply content/cloud-remote-access/using-cloud-remote-access-bundle/endpoints.md]: removed and edited edge section instead

File: content/device-management-application/monitoring-and-controlling-devices-bundle/connection-monitoring.md (hunk 2 above)
Comment (rssag) [reply content/device-management-application/monitoring-and-controlling-devices-bundle/connection-monitoring.md:52]: there was no reference to it

**Review by MWindel — APPROVED** [review]

**Pull request merged**